python/pyspark/pandas/series.py

```diff
@@ -1029,7 +1038,9 @@ def cov(self, other: "Series", min_periods: Optional[int] = None) -> float:
         if len(sdf.head(min_periods)) < min_periods:
             return np.nan
         else:
-            return sdf.select(F.covar_samp(*sdf.columns)).head(1)[0][0]
+            return sdf.select(SF.covar(F.col(sdf.columns[0]), F.col(sdf.columns[1]), ddof)).head(1)[
+                0
+            ][0]
```

Comment:
Can we make it a bit more prettier?

e.g.

```python
            return sdf.select(
                SF.covar(F.col(sdf.columns[0]), F.col(sdf.columns[1]), ddof)).head(1)[0][0]
```

Reply:
this was changed by `dev/reformat-python`...

Reply:
Yeah, sometimes `black` reformats the code looks more ugly 😂